src/calculateRentalCost.js

```diff
@@ -4,7 +4,26 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
-}
+  let result = 0;
+  const DAY_COST = 40;
+  let discount = 0;
+
+  for (let n = 1; n <= days; n++) {
+    if (n >= 7) {
+      discount = 50;
+    }
```

Comment:
Avoid using literals in expressions. Define constants for the number of days and the corresponding discounts to make the code more readable and maintainable.